Chinese/ZZB/zzb.md

```diff
@@ -11,6 +11,7 @@
 [你可知月光宝盒？](../Xinyang/MyLoveStory.md)
 
 紫霞仙子说：
-人都是失去了才懂得珍惜。
+人都是失去了才懂得珍惜。为了让至尊宝明白这个道理，紫霞打开了月光宝盒，送[至尊宝去了21世纪](../Berlixs/modernCity.md)。。。
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.